arches/app/models/migrations/7784_add_serialized_graph_to_graph_publications.py

```diff
@@ -14,7 +14,9 @@ class Migration(migrations.Migration):
     def forwards_add_serialized_graph_column_data(apps, schema_editor):
         for graph in Graph.objects.all():
             if graph.publication:
-                graph.publication.serialized_graph = JSONSerializer().serialize(graph, force_recalculation=True)
+                graph.publication.serialized_graph = JSONDeserializer().deserialize(
```

Comment:
is there any point to saving a deserialized serialized graph? Can the original graph just be used?

Reply:
No; the original graph isn't compatible with the JsonField type and blows up.

Reply:
cool just wanted to check 👍